Mathlib/Data/Vector/Basic.lean

```diff
@@ -23,6 +23,10 @@ variable {α β γ σ φ : Type*} {m n : ℕ}
 
 namespace List.Vector
 
+/-- Convert a `Vector` to an `Array`. -/
+@[deprecated toVector (since := "20/05/2025")]
+def toArray (v : Vector α n) := v.1.toArray
```

Comment:
Let's put this back in the same place rather than moving it

Reply:
OK.